packages/rpc/__tests__/formatters/result.fixtures.json

```diff
@@ -1649,6 +1649,18 @@
         "dao_type_hash": null,
         "epoch_duration_target": "0x3840",
         "genesis_hash": "0x7978ec7ce5b507cfb52e149e36b1a23f6062ed150503c85bbf825da3599095ed",
+        "hardfork_features": {
```

Comment:
https://github.com/nervosnetwork/ckb/tree/develop/rpc#method-get_consensus is update again 
"hardfork_features": [
            { "rfc": "0028", "epoch_number": "0x1526" },
            { "rfc": "0029", "epoch_number": "0x0" },
            { "rfc": "0030", "epoch_number": "0x0" },
            { "rfc": "0031", "epoch_number": "0x0" },
            { "rfc": "0032", "epoch_number": "0x0" },
            { "rfc": "0036", "epoch_number": "0x0" },
            { "rfc": "0038", "epoch_number": "0x0" },
            { "rfc": "0048", "epoch_number": null },
            { "rfc": "0049", "epoch_number": null }
         ]

Reply:
it is resolved by https://github.com/ckb-js/lumos/pull/555 and has been merged into this PR